benchmark/httpapi.py

```diff
@@ -114,10 +137,11 @@ def post(self, request):
         request.setHeader(b'content-type', b'application/json')
         try:
             json = loads(request.content.read())
+            timestamp_parser.parse(json['timestamp'])
```

Comment:
You're right, I think this would be a good change to add. I won't address it as part of this PR but I'll create an issue for it.